cmd/util/cmd/common/transactions.go

```diff
@@ -10,7 +10,7 @@ import (
 const (
 	getInfoForProposedNodesScript = `
 		import FlowIDTableStaking from 0x%s
-		pub fun main(): [FlowIDTableStaking.NodeInfo] {
+		access(all) fun main(): [FlowIDTableStaking.NodeInfo] {
```

Comment:
Maybe add line breaks after access modifiers, as they are now quite long, making the function signature hard to read

Reply:
idk, I feel like that makes the code looks pretty awkward

Reply:
I will update the tests where necessary in a follow-up PR.